nimpy/py_lib.nim

```diff
@@ -114,17 +115,57 @@ type
     PyExc_BaseException*: PPyObject # should always match any exception?
     PyExc_Exception*: PPyObject
     PyExc_ArithmeticError*: PPyObject
-    PyExc_FloatingPointError*: PPyObject
-    PyExc_OverflowError*: PPyObject
-    PyExc_ZeroDivisionError*: PPyObject
     PyExc_AssertionError*: PPyObject
-    PyExc_OSError*: PPyObject
-    PyExc_IOError*: PPyObject # in Python 3 IOError *is* OSError
-    PyExc_ValueError*: PPyObject
+    PyExc_AttributeError*: PPyObject
+    # PyExc_BlockingIOError # no
```

Comment:
I mapped many new exceptions in from https://docs.python.org/3/c-api/exceptions.html#standard-exceptions, but most of these probably will not be required as they don't directly map to a standard library nim exception as per https://nim-lang.org/docs/exceptions.html